packages/react-bootstrap-table2/src/cell.js

```diff
@@ -85,12 +93,12 @@ class Cell extends Component {
     if (cellClasses) cellAttrs.className = cellClasses;
 
     if (!_.isEmptyObject(cellStyle)) cellAttrs.style = cellStyle;
-
     if (editable && editMode !== Const.UNABLE_TO_CELL_EDIT) {
-      if (editMode === Const.CLICK_TO_CELL_EDIT) { // click to edit
-        cellAttrs.onClick = this.handleEditingCell;
-      } else { // dbclick to edit
-        cellAttrs.onDoubleClick = this.handleEditingCell;
+      const checkEditableType = _.isFunction(editable);
+      if (checkEditableType && editable(content, row, column)) {
```

Comment:
I think the editable callback function should have four argument like other cell level callback(This is for a consistency reason)
They should be `content`, `row`, `rowIndex` and `columnIndex`

Reply:
@AllenFang , where must be call callback ?? , if in here: [https://github.com/react-bootstrap-table/react-bootstrap-table2/blob/develop/packages/react-bootstrap-table2/src/row.js#L29](url), we must get content in row.js ... it's ok ?

Reply:
oh sorry, the `content` means the cell value, you can get it by `_.get(row, column.dataField)` in `row.js`